.github/workflows/ci.yml

```
@@ -413,9 +413,8 @@ jobs:
       fail-fast: true
       matrix:
         include:
-          # temporary disabled due to an issue with homebrew which is difficult to debug
-          # - target: x86_64-apple-darwin
-          #   os: macos-13
+          - target: x86_64-apple-darwin
+            os: macos-13
```

lets disable this again and move it to a new PR

```suggestion
          # temporary disabled due to an issue with homebrew which is difficult to debug
          # - target: x86_64-apple-darwin
          #  os: macos-13
```